projects/core/src/button/button.element.ts

```diff
@@ -97,7 +97,7 @@ export class CdsButton extends CdsBaseButton {
     super.firstUpdated(props);
 
     if (!this.isDefaultLoadingState(this.loadingState)) {
-      this.disabled = true;
+      super.disabled = true;
```

Comment:
The comment below says "never call this.disabled internally" and then of course I was calling this.disabled internally.